auth/src/main/java/com/firebase/ui/auth/ui/HelperActivityBase.java

```diff
@@ -70,10 +67,6 @@ public FlowParameters getFlowParams() {
         return mParams;
     }
 
-    public AuthHelper getAuthHelper() {
```

Comment:
Now the helper activity doesn't even have a helper!  Love seeing this class die haha.

Reply:
Same here, that was the first class to die when I started the refactor—it killed me when I had to add it back for the staged PRs. 😆